akka-docs-dev/rst/scala/code/docs/utils/TestUtils.scala

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.utils
+
+import java.net.InetSocketAddress
+import java.nio.channels.ServerSocketChannel
+
+object TestUtils {
+  def temporaryServerAddress(interface: String = "127.0.0.1"): InetSocketAddress = {
```

Comment:
Would it be possible to make `akka-docs-dev`: `akkaDocksDev.dependsOn(akkaActorTests % "test->test")`?
AFAIR we have this utility there